File: CodeForces/Java/CF_158A_NextRound.java
// Problem: 158A. Team
// Link: https://codeforces.com/problemset/problem/158/A

import java.util.Scanner;
class Main {
  public static void main(String[] args) {
    Scanner scan = new Scanner(System.in);
    int n = scan.nextInt();
    int k = scan.nextInt();
    int nextRound = 0;
    int[] playerScore = new int[n];
    for (int i = 0; i < n; i++) {
      int score = scan.nextInt();
      playerScore[i] = score;
    }
    scan.close();
    for (int i : playerScore) {
      if (playerScore[k-1] <= i && i>0){
        nextRound++;
      }
    }
    System.out.println(nextRound);
  }
}

File: CodeForces/Python/CF_158A_NextRound.py
# Problem: 158A. Next Round
# Link: https://codeforces.com/problemset/problem/158/A

n, k = map(int, input().split())

scores = list(map(int, input().split()))

nextRound = 0
k_score = scores[k - 1]
for score in scores:
    if score >= k_score and score > 0:
        nextRound += 1

print(nextRound)